### PR TITLE
Fixed draco recommendation

### PR DESCRIPTION
**Pull request by jahn96**
* Draco did not recommend any chart when at least one of the columns in dataset has only NaN values.
* It throws the system error to users to handle NaN's

### CODE REVIEW

File: jupyter_bifrost/bifrost.py [hunk 1]
```diff
@@ -130,10 +130,33 @@ def update_dataset(self, changes):
         )
 
     def get_data(self, df: pd.DataFrame, sampleLimit: int = None):
+        nan_columns = df.columns[df.isna().all()].tolist()
+        if len(nan_columns):
+            raise SystemError(f"{nan_columns} in this dataset have only Na's")
```
Comment (Waidhoferj) [inline jupyter_bifrost/bifrost.py:135]: "NaN values"?

File: jupyter_bifrost/bifrost.py [hunk 2]
```diff
@@ -130,10 +130,33 @@ def update_dataset(self, changes):
...
+            df = self.get_non_na(df, sampleLimit)
         return json.loads(df.to_json(orient="records"))
 
+    def get_non_na(self, df: pd.DataFrame, sampleLimit: int = None):
+        indices = self.get_each_valid_data(df)
+        if sampleLimit:
+            return pd.concat(
+                [
+                    df.loc[indices, :],
+                    df.drop(indices).sample(n=(sampleLimit - len(indices))),
```
Comment (Waidhoferj) [inline jupyter_bifrost/bifrost.py:147]: Could the `sample` call possibly grab NaN values? Not sure if this is an issue. What do you think?

**Review by Waidhoferj — CHANGES_REQUESTED** [review]
One issue with sampling. Ensure that the difference between the indices and the sample size is positive.
Screen Shot 2021-08-24 at 9 22 17 AM

File: jupyter_bifrost/bifrost.py (hunk 2 above)
Comment (jahn96) [reply jupyter_bifrost/bifrost.py:147]: It could grab NaN values. But, since we know that each column has at least one non-NaN, it shouldn't be a problem.

**Review by Waidhoferj — CHANGES_REQUESTED** [review]
One quick change

**Comment by jahn96** [comment]
* Changed it to always have at least one valid data point for each column

**Review by Waidhoferj — APPROVED** [review]
LGTM

**Pull request merged**